.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor/
 /composer.phar
 /composer.lock
+/.idea/
```

Comment:
This should be removed here and added to the global `.gitignore` file on your computer instead, because it has nothing to do with this library (but with the IDE you are using).

For more information on global `.gitignore`, see:
- https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
- http://augustl.com/blog/2009/global_gitignores/
- https://github.com/github/gitignore/tree/master/Global#globally-useful-gitignores

Reply:
Ok, I can remove this. It's just easier to click in IDE and mark as ignored :D

Reply:
It is, until you put that line into your global .gitignore. Because from then on it is ignored for every (even new) project on your computer ;)

Reply:
That also depends on particular IDE which may not recognize global git settings. But no problem I will remove it.